t/ui/10-tests_overview.t

```diff
@@ -565,23 +578,24 @@ subtest 'add comments' => sub {
       'comments created on all relevant jobs';
     is $comments->search({job_id => {-not_in => \@failed_job_ids}, text => $comment_text})->count, 0,
       'comments not created on other jobs';
-};
-
-subtest "job template names displayed on 'Test result overview' page" => sub {
-    $driver->get('/group_overview/1002');
-    is($driver->find_element('.progress-bar-failed')->get_text(), '1 failed', 'The number of failed jobs is right');
-    is($driver->find_element('.progress-bar-unfinished')->get_text(),
-        '1 unfinished', 'The number of unfinished jobs is right');
-
-    $driver->get('/tests/overview?distri=opensuse&version=13.1&build=0091&groupid=1002');
-    my @tds = $driver->find_elements('#results_DVD tbody tr .name');
-    is($tds[0]->get_text(), 'kde_variant', 'job template name kde_variant displayed correctly');
 
-    my @descriptions = $driver->find_elements('td.name a', 'css');
-    is(scalar @descriptions, 2, 'only test suites with description content are shown as links');
-    disable_bootstrap_animations;
-    $descriptions[0]->click();
-    is(wait_for_element(selector => '.popover-header')->get_text, 'kde_variant', 'description popover shows content');
+    subtest 'restart jobs with comment' => sub {
+        $driver->get('/tests/overview?state=done&result=failed');
+        $driver->find_element('button[title="Restart or comment jobs"]')->click;
+        my $comment_text = 'comment current jobs and restart';
+        my $submit_button = $driver->find_element('#restartAndCommentJobsBtn');
+        $driver->find_element_by_id('text')->send_keys($comment_text);
+        is $submit_button->get_text, 'Restart and comment on 2 jobs', 'submit button displayed with number of jobs';
+        $submit_button->click;
+        wait_for_ajax msg => 'comments created';
+        like $driver->find_element_by_id('flash-messages')->get_text, qr/Reload the page to show changes/,
+          'info about successful restart shown';
+        my @failed_job_ids = map { $_->id } $jobs->search({result => FAILED})->all;
+        is $comments->search({job_id => {-not_in => \@failed_job_ids}, text => $comment_text})->count, 0,
```

Comment:
You are only checking that comments are *not* created onother jobs, but never test that comments are created when using restart+comment.